osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/DeviceFunction.java

```diff
@@ -92,5 +94,4 @@
     UPDATE_FIRMWARE,
     UPDATE_KEY,
     START_SELF_TEST;
-    // add new DeviceFunctions in alphabetical order.
```

Comment:
I added this to the bottom as well long time ago because some people do not see the entire list on screen,

Reply:
apparently not.